libs/search/src/lib/search/search.component.html

```diff
@@ -1,9 +1,13 @@
 <p>Search:</p>
-
+<br>
 <ng-workspace05-nx-input></ng-workspace05-nx-input>
-<label>
-  <input type="checkbox" [(ngModel)]="isGlobalSearch" /> Global search?
-</label>
+<br>
+<ng-workspace05-nx-checkbox
+  [isDisabled]="false"
+  [isChecked]="isGlobalSearch"
```

Comment:
You can use two-way binding here to get rid from output and manual property update.

For more info check https://gist.github.com/gund/3ae21110e7fd9cf63cb72c54e4d6a4ac